.travis.yml

```diff
@@ -8,6 +8,16 @@ matrix:
     env:
     - LINT=true
     - COVERAGE=true
+  # Adding ppc64le jobs
+  - go: "1.13.x"
+    arch: ppc64le
+  - go: "1.14.x"
+    arch: ppc64le
+  - go: "tip"
+    arch: ppc64le
+    env:
+    - LINT=true
+    - COVERAGE=true
```

Comment:
1. please remove LINT/COVERAGE, they should be in one job only
1. is it necessary to test ppc64le on anything other than tip?

Reply:
Hi @yurishkuro 
I had removed coverage from the file. Also It will be a good idea to test ppc64le on go all go versions along with tip.

Reply:
`COVERAGE=true` must bet set only on the main Go Tip step.

> Also It will be a good idea to test ppc64le on go all go versions along with tip.

Nobody tests with "all go versions", standard practice is only the latest 2 releases. And we already do that for the default architecture, I don't see what it adds to test on another architecture. ppc64le only needs tip.

Reply:
Hi @yurishkuro ,
As suggested, I had made the changes to the file and it appears fine to me. Will you please have a look and provide suggestion. Also the changes are not adding anything extra apart from testing the existing tests on ppc64le platform.